README.md

```diff
@@ -1,23 +1,24 @@
 Equipment Requirements
 
-The following are the recommendations of the CSG for fencing equipment. Anyone who refuses to wear the proper personal protective equipment will not be permitted to participate. Anyone who intentionally obscures, or prevents from being known, errors, damages and defects in personal protective equipment may be subject to disciplinary action.
+The following are the requirements of the CSG for fencing equipment. Anyone who refuses to wear the proper personal protective equipment will not be permitted to participate. Anyone who intentionally obscures, or prevents from being known, errors, damages and defects in personal protective equipment may be subject to disciplinary action.
 
 Notes:
 
-• All references to the word “metal” mean steel unless specifically noted otherwise.
-
```

Comment:
removed because there was only one reference to "Metal", so I just changed it to steel.